app/views/doc/_ext_tutorials.erb

```diff
@@ -0,0 +1,100 @@
+        </p>
+      </li>
+      <li>
+        <h4><a href="http://git.github.io/git-reference/">Git Reference</a></h4>
+        <p class='description'>
+          The Git Reference site is meant to be a quick reference for learning and remembering the most important and commonly used Git commands.
+        </p>
+      </li>
+    </ul>
+  </div>
+  <div class='column-right'>
+    <h3>Diving Deeper</h3>
+    <ul class='content-list'>
+      <li>
+        <h4><a
+        href="https://web.archive.org/web/20161121145226/http://rypress.com:80/tutorials/git/index">Ry&rsquo;s Git
+          Tutorial</a></h4>
+        <p class='description'>
+          A hands-on introduction to the entire Git porcelain.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://web.archive.org/web/20150301060509/http://hoth.entp.com/output/git_for_designers.html">Git for Designers</a></h4>
+        <p class='description'>
+          No knowledge of version control? No problem.
+        </p>
+      </li>
+      <li>
+        <h4><a href="http://eagain.net/articles/git-for-computer-scientists/">Git for Computer Scientists</a></h4>
+        <p class='description'>
+          A quick introduction to Git internals for people who aren't scared by words like Directed Acyclic Graph.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://github.com/sensorflo/git-draw/wiki">git-draw</a></h4>
+        <p class='description'>
+          git-draw is a small tool that draws nearly the full content of a tiny git repository as a graph. It helps people with an engineering background learning Git's internals.
+        </p>
+      </li>
+      <li>
+        <h4><a href="http://www-cs-students.stanford.edu/~blynn/gitmagic/">Git Magic</a></h4>
+        <p class='description'>
+          An alternative book with the source <a href="https://github.com/blynn/gitmagic/tree/master">online</a>.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://help.github.com/">Help.GitHub</a></h4>
+        <p class='description'>
+          Guides on a variety of Git and GitHub related topics.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://www.packtpub.com/application-development/git-version-control-made-easy">Git: Version Control Made Easy</a></h4>
+        <p class='description'>
+          Use the powerful features of Git in your projects.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://www.packtpub.com/web-development/version-control-git-video">Version Control with Git [Video]</a></h4>
+        <p class='description'>
+          Your Guide to deliver great code using TFS and Git.
+        </p>
+      </li>
+      <li>
+        <h4><a href="https://try.github.io/">Git Tutorial - Try Git</a></h4>
+        <p class='description'>
+          Learn how to use Git by reviewing the basic concepts of Git version control.
+        </p>
+      </li>
+    </ul>
+  </div>
+</div>
```

Comment:
Please make sure files end with newlines. Otherwise we just end up with spurious diffs later on.